bulldozer/signals.go

```diff
@@ -21,62 +21,186 @@ import (
 	"strings"
 
 	"github.com/palantir/bulldozer/pull"
+	"github.com/pkg/errors"
 	"github.com/rs/zerolog"
 )
 
+type Signal interface {
+	Enabled() bool
+	// Determine if the signal has values assigned to it and should be considered when matching
```

Comment:
style: these should appear above the interface methods, like normal documentation comments